### ISSUE TITLE
[Cosmetic mod MoreHead] it don wroky

### ISSUE THREAD

**Issue by CatsArmy** [FluffyFishGames/AdvancedCompany#79]
Please state the used AdvancedCompany version and the mod which is incompatible or compatible.
AC version = latest
[Error  : Unity Log] TypeLoadException: Could not resolve type with token 01000025 from typeref (expected class 'MoreCompany.Utils.BundleUtilities' in assembly 'MoreCompany, Version=1.7.1.0, Culture=neutral, PublicKeyToken=null')
Stack trace:
MoreHead.PluginOne.Init () (at <a1c4596419cf4b08a6df15389a8517e9>:0)
MoreHead.PluginOne.Awake () (at <a1c4596419cf4b08a6df15389a8517e9>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()

i susspect you need to readded the BundleUtilities Class To avoid this type of errors

**Comment by Hardeh** [comment]
I think it will be easier to fix from MoreHead author's side, but he was left no means to contact him.

**Comment by CatsArmy** [comment]
yeah ik i even tried it with that classes added and it didnt fix it but i belive its not good practice to just remove a class like that if that is ment to be a replacement for the cosmetics thingy

**Comment by CatsArmy** [comment]
not a advanced company issue
